lib/api_hammer/rails/unmunged_request_params.rb

```diff
@@ -1,12 +1,13 @@
 module ApiHammer::Rails
-  # request parameters (not query parameters) without the nil/empty array munging that rails does 
+  MUTEX_FOR_THREAD = Mutex.new
```

Comment:
at first I thought an object in a constant is not good, but that puts it in the right scope

maybe `Thread.current['unmunged_params_mutex'] = Mutex.new`

Reply:
that would have literally no effect whatsoever ... ensuring that only one instance of the code can be run per thread. but the nature of a thread is that it only runs one thread of code at a time.

Reply:
À ‘$global‘ should work then


Why don't you propose solution instead of just saying it won't work?

Reply:
a global would be the same as a constant. I said in slack I am considering how a correct solution could be implemented.